app/assets/javascripts/lessons.js

```diff
@@ -67,14 +67,20 @@ function addActiveClass() {
 
 function constructLessonNavigation() {
   var commonHeadings = getLessonHeadings();
-  if (commonHeadings.length === 0) return;
-
-  var lessonNavigationHTML = lessonNavigation(commonHeadings);
-  var lessonNavigationElement = document.querySelector('.lesson-navigation');
-
-  lessonNavigationElement.innerHTML = lessonNavigationHTML;
-  addActiveClass();
-  Stickyfill.add(lessonNavigationElement);
+  if (commonHeadings.length < 2) {
+    var navigationColumn = document.querySelector('.lesson .col-lg-3');
+    var lessonColumn = document.querySelector('.lesson .row');
+
+    navigationColumn.style.display = 'none';
```

Comment:
How about using `navigationColumn.classList.add('hidden')` to stay consistent with the next line?

You'll have to define the CSS for `hidden` class or use [BS4's utility class](https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements) to hide elements.

Reply:
Sounds good, i'll take a look.